### PR TITLE
Refactoring and migration to start to work in OCA/sprint,

### PR DESCRIPTION
**Pull request by nhomar**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Nhomar, you have to rebase

**Comment by parthivgls** [comment]
Looks good. +1

**Comment by max3903** [comment]
@nhomar Can you give us a chance to review ?

File: README.md [hunk 1]
```diff
@@ -1,9 +1,188 @@
-[![Build Status](https://travis-ci.org/OCA/vertical-medical.svg?branch=8.0)](https://travis-ci.org/OCA/vertical-medical)
-[![Coverage Status](https://coveralls.io/repos/OCA/vertical-medical/badge.png?branch=8.0)](https://coveralls.io/r/OCA/vertical-medical?branch=8.0)
+What is Odoo Medical
```
Comment (max3903) [inline README.md:1]: Can you keep the badges and use the svg format instead of png ?

**Pull request merged**